dev/test-fpindex-api.py

```
@@ -0,0 +1,196 @@
+import json
+import msgpack
+import sys
+
+FPINDEX_URL = "http://localhost:6081"
+INDEX_NAME = "test"
+
+def test_health():
+    """Test health endpoint"""
+    print("Testing health endpoint...")
+    try:
+        resp = requests.get(f"{FPINDEX_URL}/_health")
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)